cli/cmd_root.go

```diff
@@ -48,6 +49,7 @@ func init() {
 	buildCmd.Flags().Bool("dummy", false, "Create a dummy bobfile")
 	buildCmd.Flags().Bool("no-cache", false, "Set to true to not use cache")
 	buildCmd.Flags().Bool("insecure", false, "Set to true to use http instead of https when accessing a remote artifact store")
+	buildCmd.Flags().StringSliceVar(&flagEnvVars, "env", []string{}, "Set environment variables to build task")
```

Comment:
Should also work for the run command

Reply:
I created a separate issue for that https://github.com/benchkram/bob/issues/134